clang/test/CodeGen/LoongArch/inline-asm-gcc-regs.c

```diff
@@ -5,6 +5,7 @@
 
 // CHECK-LABEL: @test_r0
 // CHECK: call void asm sideeffect "", "{$r0}"(i32 undef)
+// CHECK: call void asm sideeffect "", "{$r0}"(i32 undef)
```

Comment:
Maybe we can use [the `CHECK-COUNT` directive](https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-count-directive): `CHECK-COUNT-2` and a `CHECK-NOT`, to ensure there are exactly 2 lines having this content.

Otherwise LGTM, thanks!